### PR TITLE
CAS-599: Check if user meets requirements to create a proposal

### PR DESCRIPTION
**Pull request by germanurrus**
- Adds custom theme updates for Chakra
- Implements custom component using Chakra

Screen Shot 2022-10-24 at 19 39 50

### CODE REVIEW

File: frontend/packages/client/src/helpers/theme/index.js [hunk 1]
```diff
@@ -1,14 +1,44 @@
...
+    Card: CardStyle,
+    Link,
+    Text,
+    // Other components go here
   },
-});
+};
 
-export default theme;
+export default extendTheme(overrides);
```
Comment (germanurrus) [inline frontend/packages/client/src/helpers/theme/index.js:44]: this file provides configuration to use a custom theme with chakra

`definitions.js` file overwrites default definitions by Chakra to match designs
`components` folder contains one file per component we are overwriting to match designs, by having one file per component it's easy to find and keeps things more isolated 
`foundations` folder contains definitions for borders, radius, typography (currently we are using bulma and it should be ported here)
`styles` file defines global styles for html elements
there's more to add but this is the basic structure

File: frontend/packages/client/src/components/Card/styles.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      minWidth: '210px',
+      height: '69px',
+    },
+  },
+  defaultProps: {
+    background: 'white',
+  },
+};
+
+export default Card;
```
Comment (germanurrus) [inline frontend/packages/client/src/components/Card/styles.js:37]: this is a custom component, therefore defining the style here makes it easier to identify it as a custom component and not mixing this definition with the theme overwrites.

File: frontend/packages/client/src/components/Card/Card.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Box, useStyleConfig } from '@chakra-ui/react';
+
+export default function Card(props) {
+  const { variant, ...rest } = props;
+
+  const styles = useStyleConfig('Card', { variant });
+
+  // Pass the computed styles into the `__css` prop
+  return <Box __css={styles} {...rest} />;
+}
```
Comment (germanurrus) [inline frontend/packages/client/src/components/Card/Card.js:10]: Custom component

**Review by jbluks — APPROVED** [review]
LGTM!

**Review by jbluks — CHANGES_REQUESTED** [review]
Can we confirm that these changes are meant for release-v1.6.0 before merging? (Confirmed)

**Review by jbluks — APPROVED** [review]

**Review by jacksonConrad — APPROVED** [review]

**Pull request merged**